bindings/matlab/Visualization/vis_iCub_test.m

```diff
@@ -0,0 +1,38 @@
+icubModelPath=[ getenv('ROBOTOLOGY_SUPERBUILD_SOURCE_DIR') '/build-linux/install/share/iCub/robots/iCubGazeboV2_5/'];
```

Comment:
Please avoid this kind of absolute references that could  change depending on the user specific setup. If this is not possible, feel free to remove the test.

Reply:
I think that for avoiding that I would need to add a custom urdf with the meshes inside the testing or example folder

Reply:
If  it is just an example, you can just document that this example depends on having icub-models installed, and you can have a variable that the user should replace with the installation prefix of icub-models on its machine.

Reply:
This file was eliminated